recipe/meta.yaml

```
@@ -29,8 +29,9 @@ requirements:
     - aws-c-event-stream 0.2.15
     - aws-crt-cpp 0.18.16
     - aws-checksums 0.1.13
-    - libcurl 7.88.1
+    - libcurl 8.9.1
```

```suggestion
    - libcurl {{ libcurl }}
```
This increases the range of curl versions installable with this package.

---

There are a few medium and one severe vulnerabilities in versions <8.2.0 https://curl.se/docs/vulnerabilities.html
Our current lubcurl is 8.1.1 https://github.com/AnacondaRecipes/aggregate/blob/master/conda_build_config.yaml#L134

---

Sure, but building with what is in the cbc will still resolve in the latest compatible on main.
In host: 8.1.1
In run: >=8.1.1,<9.0
Resolved when installed: 8.9.1

---

Fixed